src/lib/actionrunner.js

```diff
@@ -36,6 +36,7 @@ const OPENWHISK_DEFAULTS = {
         // "nodejs:8"   : "openwhisk/action-nodejs-v8:latest",
         "nodejs:10"  : "adobeapiplatform/adobe-action-nodejs-v10:3.0.22", // see: https://hub.docker.com/r/adobeapiplatform/adobe-action-nodejs-v10/tags
         "nodejs:12"  : "adobeapiplatform/adobe-action-nodejs-v12:3.0.22", // see: https://hub.docker.com/r/adobeapiplatform/adobe-action-nodejs-v12/tags
```

Comment:
Maybe update these as well to their latest version.

Seems to be `3.0.27` for both v10 and v12.

https://hub.docker.com/r/adobeapiplatform/adobe-action-nodejs-v10/tags
https://hub.docker.com/r/adobeapiplatform/adobe-action-nodejs-v12/tags